api/src/main/java/com/codeforcommunity/exceptions/FailedFileLoadException.java

```diff
@@ -0,0 +1,23 @@
+package com.codeforcommunity.exceptions;
+
+import com.codeforcommunity.rest.FailureHandler;
+import io.vertx.ext.web.RoutingContext;
+
+public class FailedFileLoadException extends HandledException {
```

Comment:
I had time so I just implemented a custom exception as you mentioned last night! @jackblanc